source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1195,6 +1196,56 @@ namespace
         }
     }
 
+    void writeMasterPrim(DocumentPtr document, RtStagePtr rtStage, const RtPrim& masterPrim)
+    {
+        if (!masterPrim.isValid())
+        {
+            return;
```

Comment:
Should this return a bool or throw an exception? I would think the caller of this function would be surprised by this result, no?